internal/server/data/data.go

```diff
@@ -105,6 +105,7 @@ func getDefaultSortFromType(t interface{}) string {
 }
 
 func get[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (*T, error) {
+	db = db.Session(&gorm.Session{})
```

Comment:
None of the fields on this `gorm.Session` struct has any godoc, so I have no idea which ones should be set. I'm guessing none, since it clones the config of the original `gorm.DB`.